docs/website/blog/2022-06-14-binding-external-service.md

```diff
@@ -0,0 +1,187 @@
+
+You can find information about the Service Binding Operator [here](/docs/getting-started/cluster-setup/kubernetes#optional-installing-the-service-binding-operator).
+
+A ServiceBinding defines a binding between an *Application* and a *Service*. The credentials injected into the application
+can be defined in diffent ways:
+- if the service is an Operator-backed service running on the cluster, the details of the injected credentials can be set
+as annotations of the CRD associated with the Operator-backed service,
+- in any case, the details of the injected credentials can be set in the resource itself (not the CRD). 
+- in any case, the details of the injected credentials can be set in the ServiceBinding resource itself.
+
+In this article, we are not using an Operator-backed service, but an external service referenced by a Service resource.
+As the Service resource is a native Kubernetes resource, we cannot add annotations to its CRD, so we will add annotations to
+the Service resource itself.
+
+You can modifiy the definition of the Service, by adding the following annotations:
+
+```
+kind: Service
+apiVersion: v1
+metadata:
+  name: redis
+  annotations:
+    service.binding/host: path={.metadata.name}.{.metadata.namespace}.svc.cluster.local
+    service.binding: path={.metadata.name}-credentials,objectType=Secret
+spec:
+  type: ClusterIP
+  ports:
+  - port: 6379
+    targetPort: 6379
+```
+
+In this snippet, the first annotation
+
+```
+service.binding/host: path={.metadata.name}.{.metadata.namespace}.svc.cluster.local
+```
+
+indicates to the SBO to inject a `host` variable into the application, with a value computed based on the
+`metadata.name` and `metadata.namespace` of the Service resource. In this example, the value `redis.external-services.svc.cluster.local`
+will be given to the `host` variable.
+
+
+The second annotation
+
+```
+service.binding: path={.metadata.name}-credentials,objectType=Secret
+```
+
+indicates to the SBO to inject the values defined in the Secret, whose name is the name of the Service resource
+followed by `-credentials` (`redis-credentials` in our example), into the application. In this example, the variable `password` 
+with a value `MyEasyPassword` will be injected into the applications's Pod.
+
+## Adding a ServiceBinding to the Devfile
+
+To define a ServiceBinding, we need information (group, version, kind, name and namespace) about the Application and the Service.
+
+In our example, the service is a Kubernetes Service (group "", version "v1" and kind "Service") named `redis`
+in the `external-services` namespace, 
+
+The application will be the Deployment resource (group "apps", version "v1", kind "Deployment") created by odo when you run `odo dev`.
+By convention, the Deployment name will be the name of the Devfile (in the `.metadata.name` field) followed by `-app` (`my-nodejs-app-app` in our example).
```

Comment:
You are mentioning `my-nodejs-app-app`, but I didn't see it mentioned before.

Reply:
Fixed